ClientTests/TPStatsBlocklistsTests.swift

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@testable import Client
+
+import XCTest
+
+class TPStatsBlocklistsTests: XCTestCase {
+    var blocklists: TPStatsBlocklists!
+    
+    override func setUp() {
+        super.setUp()
+        
+        blocklists = TPStatsBlocklists()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        blocklists = nil
+    }
+    
+    func testLoadPerformance() {
+        self.measureMetrics([.wallClockTime], automaticallyStartMeasuring: true) {
+            blocklists.load()
+            self.stopMeasuring()
+        }
+    }
+    
+    func testURLInListPerformance() {
+        blocklists.load()
+        
+        let whitelistedRegexs = [".google.com"].compactMap { (domain) -> NSRegularExpression? in
+            return wildcardContentBlockerDomainToRegex(domain: domain)
+        }
+        
+        self.measureMetrics([.wallClockTime], automaticallyStartMeasuring: true) {
+            for _ in 0..<100 {
```

Comment:
This will run 1000x, how long does it take? Preferred to keep perf measurements in the 100ms range to keep test suites running quickly.

Reply:
Avg 0.003, though I think the cumulative start-to-finish test case (including setup, load, test,  teardown) takes 1.2s on my machine. Setting the loop to `0..<1` runs start-to-finish 1.15s but the avg is 0.000.

Reply:
A <50ms test SGTM, thanks for clarifyling!